src/commands/init.js

```diff
@@ -15,17 +15,20 @@
  */
 import inquirer from 'inquirer';
 import fs from 'fs-extra';
-import chalk from 'chalk';
 import path from 'path';
 import _ from 'lodash';
+import chalk from 'chalk';
 
 import { controller } from '../api';
 import * as config from '../config';
+import { createCommand } from '../util';
 import * as template from '../template';
+import LoginCommand from './login';
 
 function ensureLoggedIn(argv) {
   if (!argv.currentAccount) {
-    return Promise.reject('Requires authentication, please run skycli login.');
+    console.log(chalk.yellow('Requires authentication, please run login.'));
```

Comment:
I think `Requires authentication, please login.` is better, since we prompt for email immediately.

Reply:
oh... forgot to delete that word